lbry/lbry/conf.py

```diff
@@ -159,8 +162,8 @@ def _parse_list(l):
         return {'amount': amount, 'currency': currency}
 
     def deserialize(self, value):
-        if value is None:
-            return
+        if not value:
+            return None
```

Comment:
Why this change?

Reply:
To support the empty string case, which we don't want to do, see below; I'll revert it.